operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Domain.java

```diff
@@ -821,6 +822,12 @@ private void verifyIntrospectorJobName() {
       }
     }
 
+    private void verifyCommonMountContainers() {
+      // if the common mount is specified, verify that at least one container is defined.
```

Comment:
serverPod can scope to the cluster and pod level, is this the place to check those too?